waspc/data/Generator/templates/sdk/wasp/client/webSocket/index.ts

```diff
@@ -5,6 +5,9 @@ import type {
   ServerToClientEvents,
 } from 'wasp/server/webSocket'
 
+// PUBLIC API
+export type { WebSocketClientConfigFn } from './WebSocketProvider'
```

Comment:
Don't forget to update the import table after merge (I won't share the URL here since it's public).